scrapy/czlscrape/settings.py

```diff
@@ -89,3 +89,7 @@
 #HTTPCACHE_DIR = 'httpcache'
 #HTTPCACHE_IGNORE_HTTP_CODES = []
 #HTTPCACHE_STORAGE = 'scrapy.extensions.httpcache.FilesystemCacheStorage'
+
+LOG_LEVEL = 'WARN'
+LOG_FORMAT = '%(asctime)s srv="czl-scrape" [%(thread)d] %(levelname)s %(name)s: %(message)s'
```

Comment:
> maybe use %(funcName)s instead of name?

Maybe both? `%(name)s` is the name of the module (e.g. `czlscrape.spiders.dialog`), so it's quite important. `%(funcName)s` is just the name of the function, e.g. `parse`.

`2017-04-06T11:23:40+0300 srv="czl-scrape" [140735236087808] INFO czlscrape.spiders.dialog parse: some log message`